third_party/move/move-vm/types/src/delayed_values/bcs_utils.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// TODO[agg_v2](cleanup): deduplicate against aptos-types an figure out where
```

Comment:
and

Reply:
move-core-types might be a good place for low level helpers, but if I remember correctly we also have uleb helpers in move-binary-format.

Reply:
Moved these to binary-format helpers (indeed, there were some already there, thanks for pointing out!)